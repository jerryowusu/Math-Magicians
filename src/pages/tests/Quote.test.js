/* eslint-disable jsx-quotes */
import React from 'react';
import renderer from 'react-test-renderer';
// import { BrowserRouter as Router } from 'react-router-dom';
import Quotes from '../Quote';

it('should display the home page content 1', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
