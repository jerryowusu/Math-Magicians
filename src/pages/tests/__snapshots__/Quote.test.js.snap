// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the home page content 1 1`] = `
<div
  className="container"
>
  <h3>
    Mathematics is the result of mysterious powers which no one understands, and which the unconscious recognition of beauty must play an important part. Out of an infinity of designs a mathematician chooses one pattern for beauty’s sake and pulls it down to earth. — Marston Morse, American mathematician
  </h3>
</div>
`;

exports[`should display the home page content 1 2`] = `
  <div className="container">
    <h3>
      Mathematics is the result of mysterious powers which no one understands,
      and which the unconscious recognition of beauty must play an important
      part. Out of an infinity of designs a mathematician chooses one pattern
      for beauty’s sake and pulls it down to earth. — Marston Morse, American
      mathematician
    </h3>
  </div>
`;
